.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "window.title": "${activeEditorShort}${separator}${rootName}${separator}${profileName}${separator}[${activeRepositoryBranchName}]",
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  // If you are using `sysoev.vscode-open-in-github` extension, make sure to enable this setting
+  // "openInGitHub.defaultBranch": "unstable",
```

Comment:
why is this commented out? If I enable it I still have this file modified locally which is not good